serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -8,6 +8,37 @@ toc::[]
 
 If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest version.
 
+[id="update-api-group_{context}"]
+== Updating the API group
+
+In {ServerlessProductName} 1.4.0 and newer versions, the `KnativeServing` object's API group changed from `serving.knative.dev` to `operator.knative.dev`.
+You must adjust any of your scripts or applications that rely on the old API group to use the new API group.
+
+Once you have updated references to the new API group, you must remove any older custom resource (CR) versions and use the newly deployed `KnativeServing` CR instead.
+
+To safely do this without downtime, remove the owner reference from the newly deployed `KnativeServing` CR by entering the following command:
+----
+ $ oc edit knativeserving.operator.knative.dev knative-serving -n knative-serving
+----
+
+After the owner reference has been removed, you can safely remove any older CR versions and start using the new one.
```

Comment:
Is there any detriment to removing the old CR before you fix all these references? Is there a method for finding which ones to update? (If so, and it's not covered elsewhere, consider adding that here.)

Reply:
cc @markusthoemmes wdyt?

Reply:
Yeah if you delete before you remove the owner reference, everything will get deleted because of the ref being there. There's unfortunately no way of querying that really.

You could consider doing: `oc get knativeserving.operator.knative.dev --all-namespaced` to find them all and make sure none have any ownerReferences left.

Reply:
After discussions in Slack, I really don't think there's a need to add this step re `oc get knativeserving.operator.knative.dev --all-namespaced`. It refers to just listing all new CRs, which there should only be one of anyway, so it seems like it will just add confusion for users.